### DISCUSSION TITLE
Popups in v4

### DISCUSSION THREAD

**Discussion by schoier** [microsoft-search/pnp-modern-search#2082] [Ideas]
Hi, 
is it possible to create popups in a custom search result template as in v3 (https://microsoft-search.github.io/pnp-modern-search/v3/search-parts/templating/#web-components)?

**Comment by wobba** [comment]
The pop-up component has not been added to v4, but a good suggestion 👍

**Reply by schoier** [reply]
Thanks for a quit reply!

**Reply by furler** [reply]
Is this on the roadmap at all? I'd love to implement it as well.

**Reply by kasperbolarsen** [reply]
@furler the project would very much appreciate a PR from you implementing this feature 😁
